### ISSUE TITLE
After Nuget Update doesn't start

### ISSUE THREAD

**Issue by romanWienicke** [MicrosoftDocs/bot-docs#834]
After updating all packages to the latest version the bot project does not start. To reproduce this issue just create a empty "Echo Bot" from the template and start it in Visual Studio 2017 with the "Start" Button (debugging, IIS Express). Every thing should work as expected. Now got to "Manage Nuget Packeges" and update all packages. When you try to start the project now it fails with the message "HTTP Error 502.5 - ANCM Out-Of-Process Startup Failure". Changing the debug settings does not have any impact.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9d5bd35d-3178-42cd-2893-3b6a4b7a94ce
* Version Independent ID: 1545fd44-d471-d947-f94d-4b5e8d6e853f
* Content: [Tutorial to create and deploy a basic bot - Bot Service](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-tutorial-basic-deploy?view=azure-bot-service-4.0&tabs=csharp)
* Content Source: [articles/v4sdk/bot-builder-tutorial-basic-deploy.md](https://github.com/MicrosoftDocs/bot-docs/blob/live/articles/v4sdk/bot-builder-tutorial-basic-deploy.md)
* Service: **bot-service**
* Product: **bot-framework**
* GitHub Login: @ivorb
* Microsoft Alias: **v-ivorb**

**Comment by cleemullins** [comment]
Could you file in in the primary Github repo for .Net? 
That repo is: https://github.com/Microsoft/botbuilder-dotnet

Once it's there, I can ask the support team to engage. Nothing about what you're doing seems weird. 

Could the bug you're seeing be related to this, which [I see on Stack Overflow](https://stackoverflow.com/questions/54020747/ancm-inprocess-startup-failed-because-of-invalid-runtimeconfig-json).

**Comment by romanWienicke** [comment]
Hi,
I published my testproject on Github: https://github.com/romanWienicke/QandABot/tree/master/QandABot

I don't think it's relataed to the bug you mentioned Stack Overflow<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fstackoverflow.com%2Fquestions%2F54020747%2Fancm-inprocess-startup-failed-because-of-invalid-runtimeconfig-json&data=02%7C01%7C%7Ca009945179bd4761fb4e08d68eccbe3e%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636853407349346706&sdata=DFnr8loTLw8wjA%2BS2qFUfoiAEEbV49eR1XHZZMNwkAI%3D&reserved=0>.

To reproduce the Issue just create an empty Echo Bot and update to .net core 2.2. (everything is working..)
Now add the Microsoft.Bot.Builder.AI.QnA nuget package - here there are some issues with possible downgrading of nuget packages and the project does not start anymore.


best regards,
Roman
________________________________
Von: Chris Mullins <notifications@github.com>
Gesendet: Samstag, 9. Februar 2019 21:25
An: MicrosoftDocs/bot-docs
Cc: Roman Wienicke; Author
Betreff: Re: [MicrosoftDocs/bot-docs] After Nuget Update doesn't start (#834)


Could you file in in the primary Github repo for .Net?
That repo is: https://github.com/Microsoft/botbuilder-dotnet<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoft%2Fbotbuilder-dotnet&data=02%7C01%7C%7Ca009945179bd4761fb4e08d68eccbe3e%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636853407349336695&sdata=RLcZp0U0Pj%2FGxrJZSgMVtisnoWEruqhJtF1njv%2Bi%2F8Q%3D&reserved=0>

Once it's there, I can ask the support team to engage. Nothing about what you're doing seems weird.

Could the bug you're seeing be related to this, which I see on Stack Overflow<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fstackoverflow.com%2Fquestions%2F54020747%2Fancm-inprocess-startup-failed-because-of-invalid-runtimeconfig-json&data=02%7C01%7C%7Ca009945179bd4761fb4e08d68eccbe3e%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636853407349346706&sdata=DFnr8loTLw8wjA%2BS2qFUfoiAEEbV49eR1XHZZMNwkAI%3D&reserved=0>.

**Comment by ivorb** [comment]
Hi Roman, so your issue is with adding the QnA package? Did you add an issue the dotnet repo @cleemullins linked above? I didn't see it there, but might have missed it.

**Comment by romanWienicke** [comment]
No, I didn't add it to @cleemullins.

**Comment by ivorb** [comment]
This is now tracked in the .NET repo, as referenced above.

**Comment by EricDahlvang** [comment]
@romanWienicke I downloaded your GitHub repo, and the bot starts without issue from Visual Studio 2017.  This must be something with your local environment.

**Comment by EricDahlvang** [comment]
@romanWienicke A few debug tip ideas from drub0y here: https://github.com/Microsoft/botbuilder-dotnet/issues/1377#issuecomment-463422652

**Comment by romanWienicke** [comment]
@EricDahlvang thanks for testing, it seems as the issue is gone with the latest updates. I just tried to reproduce my issue and everything was working. 
As before it created a new "Echo Bot" from the Bot Service Template. The nuget references seem to have changed, because after creating the project there where no Updates. Before there where 9 Updates needed (see screenshot below - an older Bot created last week).
![image](https://user-images.githubusercontent.com/3491656/52766645-13c4cb00-3028-11e9-9e5c-46a5c1723c11.png)


As I created the same "Echobot" today there where no nuget updates needed (see screenshot beleow - 0 -Updates). It seems as there where some updates in the themplate - but now everything seems to work fine now!
![image](https://user-images.githubusercontent.com/3491656/52766589-ec6dfe00-3027-11e9-9382-6d65a7357362.png)

Thank you
Roman
